docs/modules/ROOT/pages/installing.adoc

```diff
@@ -4,26 +4,61 @@
 
 Micrometer contains a core library with the instrumentation SPI and an in-memory implementation that does not export data anywhere, a series of modules with implementations for various monitoring systems, and a test module.
 
-To use Micrometer, add the dependency for your monitoring system.
+To use Micrometer, add the dependency for your monitoring system. It is recommended to use the BOM provided by Micrometer, you can configure it as follows (you only need to declare it once in your project).
 
-The following example adds Prometheus in Gradle:
+NOTE: If you use a framework, it might already come with a BOM that defines the Micrometer versions or imports the Micrometer BOM. If this is the case (for example if you use Spring Boot), you can skip configuring the Micrometer BOM.
```

Comment:
```suggestion
NOTE: If you use a framework, it might have dependency management that defines Micrometer versions or imports the Micrometer BOM. You can defer to the framework's dependency management instead of declaring the Micrometer BOM directly in that case.
```
Reworded it some. It is possible to use Spring Boot without its dependency management. Initially I tried to rewrite to specifically say if you're using Spring Boot's dependency management, but I didn't want to get into the different ways that might happen (Gradle plugin vs platform BOM vs Maven parent etc), and it felt redundant with the more generic "framework's dependency management"